src/components/AddTodoForm.js

```diff
@@ -37,6 +42,9 @@ function AddTodoForm({ onAddTodo }) {
 
 };
 
+AddTodoForm.propTypes = {
+    onAddTodo: PropTypes.func,
```

Comment:
great job! :100: 
Here you can also ensure that the function onAddTodo is required when using this component.
`onAddTodo: PropTypes.func.isRequired`